### ISSUE TITLE
params number?

### ISSUE THREAD

**Issue by lerogo** [zhangxinsong-nlp/XFM#2]
Hi,
Thank you for your excellent work and elegant code. However, I have noticed that the size of the XFM base model parameters is incorrect. My result is 403,922,165.
Additionally, in your paper, CLIP's parameter count is reported as 490M. Could you please verify this? Thank you.

Regards,

**Comment by lerogo** [comment]
[params.txt](https://github.com/zhangxinsong-nlp/XFM/files/12081407/params.txt)

**Comment by xszhang0320** [comment]
hi,
Thank you for your attention. "roberta.embeddings" and "lm_cap_head" are not actually used in the fusion encoder. Therefore, they are excluded from the params statistics. We will update the specific param number of ViT-L/14 in our revised version. Thank you for your kind reminder.

**Comment by lerogo** [comment]
Hi,
I calculated the parameter count of the CLIP (ViT-L/14@336px) model to be 429M, which is significantly different from the value you reported. However, I haven't found any other articles mentioning the parameter count of CLIP. Could you help me verify it?🌹

**Comment by xszhang0320** [comment]
yeah, you are right. We will fix that in the revised version. We mistook ALIGN's param number as CLIP because they are basically the same model architecture. I want to thank you again for your reminder.

**Comment by lerogo** [comment]
thank you very much!

**Comment by lerogo** [comment]
hi,
If "roberta.embeddings" and "lmcaphead" are not actually used, why are they added to the optimizer?
